docker/README.md

```diff
@@ -357,31 +350,31 @@ curl -X GET http://your_public_host_name/ping
 ### Stopping the current MMS instance
 To stop the MMS running inside the docker instance, run the following
 ```bash
-$ docker exec mms bash -c "mxnet-model-server.sh stop"
+$ docker exec mms bash -c "mxnet-model-server stop"
 ```
 
 ### Usage help for MMS
 ```bash
-docker exec mms bash -c "mxnet-model-server.sh help"
+docker exec mms bash -c "mxnet-model-server help"
 ```
 ```text
 Usage:
 
-/mxnet_model_server/mxnet-model-server.sh [start | stop | restart | help] [--mms-config <MMS config file>]
+mxnet-model-server [start | stop | restart | help] [--mms-config <MMS config file>]
 
 start        : Start a new instance of MXNet model server
 stop         : Stop the current running instance of MXNet model server
 restart      : Restarts all the MXNet Model Server worker instances
-help         : Usage help for /mxnet_model_server/mxnet-model-server.sh
+help         : Usage help for /mxnet_model_server/mms
```

Comment:
Isn't this supposed to be /mxnet-model-server and not /mms?

Reply:
True. I missed this. Redid this.